modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -890,7 +893,8 @@
               "STOP": "Stop execution workflow instance",
```

Comment:
Not your PR directly but can you maybe fix this as well? Either change it to `Stop execution of workflow instance` or to `Stop executing workflow instance`. Furthermore, you started removing the word instance at some places. If you want to remove it consistently, these keys could get be without as well: `Stop executing workflow` or even just `Stop workflow`

Reply:
"Stop workflow" seems reasonable.